presto-parser/src/main/java/com/facebook/presto/sql/tree/SimpleGroupBy.java

```diff
@@ -42,7 +42,7 @@ public SimpleGroupBy(NodeLocation location, List<Expression> simpleGroupByExpres
     private SimpleGroupBy(Optional<NodeLocation> location, List<Expression> simpleGroupByExpressions)
     {
         super(location);
-        this.columns = requireNonNull(simpleGroupByExpressions);
+        this.columns = ImmutableList.copyOf(requireNonNull(simpleGroupByExpressions, "simpleGroupByExpressions is null"));
```

Comment:
This style is all over the code, so if we want to move away from this, we should change the existing code so users don't follow the "examples" in the code.